src/compas_fea/__init__.py

```diff
@@ -30,7 +30,7 @@
 __copyright__ = 'Block Research Group'
 __license__ = 'MIT License'
 __email__ = ''
-__version__ = '0.3.0'
+__version__ = '0.4.0'
```

Comment:
are you locally already at `0.4.0`?
bumping the version should happen with `invoke` such that it is done throughout the package automatically...